File: 0965-unique-email-addresses/0965-unique-email-addresses.py
class Solution(object):
    def numUniqueEmails(self, emails):
        """
        :type emails: List[str]
        :rtype: int
        """
        unque_set=set()
        for email in emails:
            name,domain=email.split('@')
            name= name.split('+')[0]
            name=name.replace('.','')
            unque_set.add((name,domain))
        return len(unque_set)
            
        

File: 0965-unique-email-addresses/0965-unique-email-addresses.js
/**
 * @param {string[]} emails
 * @return {number}
 */
var numUniqueEmails = function(emails) {
    let emailSet=new Set()
    for(let email of emails){
        let emailDomain= email.split('@')
        let name=emailDomain[0]
        let domain= emailDomain[1]
        name=name.split('+')[0]
        name=name.replaceAll('.','')
        let unique=name+'@'+domain
        emailSet.add(unique)

    }
    return emailSet.size
};